db_revisions/versions/5a775dd75356_create_filing_period_table.py

```diff
@@ -20,12 +20,12 @@
 def upgrade() -> None:
     op.create_table(
         "filing_period",
-        sa.Column("id", sa.INTEGER, primary_key=True, autoincrement=True),
         sa.Column("name", sa.String, nullable=False),
```

Comment:
I believe we were talking about having a `code` column as the pk; we can either rename the `name` to `code`, or `code` would instead replace what used to be `id`, still having it as string / varchar; and `name` could remain the same, or become maybe `description`. e.g. {code: 2024, name/description: 2024 annual filing}, {code: 2024Q1, name/description: 2024 quarter 1 filing}. This would also drive the front end drop down, so selection value would be the code, and the display would be name/description.